patches/krita/AppData.patch

```diff
@@ -33,8 +33,8 @@ index d02fb68..62ba534 100644
    </provides>
    <content_rating type="oars-1.1"/>
    <releases>
--    <release date="2022-04-27" version="5.0.6">
-+    <release date="2022-04-27" version="5.0.6"/>
+-    <release date="2022-05-25" version="5.0.8">
```

Comment:
This one should add the entry, not replace the existing one. We should also push that upstream.

Reply:
Thanks for the catch, going to fix that patch. Looks like the 5.1 builds coming up next don't have the missing slash, but sent a PR earlier for the 5.0 branch.

Reply:
@travier I do notice Krita has multiple different AppData files. The one on their main branch has multiple versions. https://invent.kde.org/graphics/krita/-/blob/master/krita/org.kde.krita.appdata.xml#L560 but under the branch for each version it seems to only contain the latest release for that branch. https://invent.kde.org/graphics/krita/-/blob/krita/5.1/krita/org.kde.krita.appdata.xml#L516.

Reply:
Well they should not do that AFAIK. Each version should include all of the previous ones. Can you make an upstream bug report and/or the send a PR?

Reply:
Is there a link to a documentation page that mentions to include prior version that can be used as reference for a PR/bug report?

Reply:
https://freedesktop.org/software/appstream/docs/chap-Metadata.html#tag-releases. It's not mandatory, but preferred.